estuary_updater/handlers/errata.py

```diff
@@ -67,3 +90,44 @@ def handle(self, msg):
         for bug in bugs:
             bug = BugzillaBug.get_or_create({'id_': bug['bug']['id']})[0]
             advisory.attached_bugs.connect(bug)
+
+    def builds_added_handler(self, msg):
+        """
+        Handle an Errata tool activity status message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        self.koji_session = koji.ClientSession(self.config['estuary_updater.koji_url'])
+
+        nvr = msg['body']['headers']['brew_build']
+
+        try:
+            koji_build = self.koji_session.getBuild(nvr, strict=True)
+        except Exception:
+            log.error('Failed to get brew build with NVR {0}'.format(nvr))
+            raise
+
+        build_params = {
+            'completion_time': koji_build['completion_time'],
+            'creation_time': koji_build['creation_time'],
+            'epoch': koji_build['epoch'],
+            'id_': koji_build['id'],
+            'name': koji_build['package_name'],
+            'release': koji_build['release'],
+            'state': koji_build['state'],
+            'version': koji_build['version']
+        }
+
+        advisory = Advisory.get_or_create({
+            'id_': msg['body']['headers']['errata_id']
+        })[0]
+
+        build = KojiBuild.create_or_update(build_params)[0]
```

Comment:
Optional: For readability, I'd prefer if the creation of `build` was right after `build_params` was defined. That way you can see what `build_params` is used for right away.